template/studio/schema/documents/news/index.js

```diff
@@ -36,36 +43,21 @@ export default {
     },
     {
       name: 'meta',
-      title: 'Meta',
+      title: 'SEO Settings',
```

Comment:
pretty sure that re-naming it but keeping the variable name will be suuuper confusing later. how much effort would it be to properly re-name it?